File: LeetCode/35. Search Insert Position.cs
// 35. Search Insert Position
// https://leetcode.com/problems/search-insert-position/
// by Equim on 16-12-07

public class Solution {
    public int SearchInsert(int[] nums, int target) {
        int index = -1;
        foreach(var item in nums)
        {
            index++;
            if(item>=target)
                return index;
        }
        return index+1;
    }
}

File: LeetCode/35. Search Insert Position.c
// 35. Search Insert Position
// https://leetcode.com/problems/search-insert-position/
// by Equim on 16-12-07

int searchInsert(int* nums, int numsSize, int target) {
    int i;
    for(i=0;i<=numsSize-1;i++)
    {
        if(nums[i]>=target)
            return i;
    }
    return i;
}
